build/gulpfile.hygiene.js

```diff
@@ -64,6 +64,7 @@ const indentationFilter = [
 
 	// except multiple specific files
 	'!**/package.json',
+	'!**/package-lock.json',
```

Comment:
we don't need a package-lock. we use yarn as our package manager not yarn.

Reply:
i couldnt commit without adding the line, this is for linting